### ISSUE TITLE
[Discussion] By default, `distrobox rm` can be destructive when combined with a mounted home directory

### ISSUE THREAD

**Issue by AdamIsrael** [89luca89/distrobox#894]
> Is your feature request related to a problem? Please describe.

Scenario: `distrobox rm` can delete the directory mounted in the container via `--home`.

```
$ distrobox rm -f acme
Do you want to remove custom home of container acme (/var/home/stone/Clients/ACME)? [y/N]: y
acme
Successfully removed /var/home/stone/Clients/ACME
```

I would make the opinionated argument that this is a dangerous prompt to display by default.

> Describe the solution you'd like

There is a `--rm-home` flag. In my mind, that flag should be required before prompting the user to remove the custom home directory. It's a destructive operation, so all caution should be taken to ensure the user really wants that to happen.

In other words, I think the command should look like this:

```
$ distrobox rm -f --rm-home acme
Do you want to remove custom home of container acme (/var/home/stone/Clients/ACME)? [y/N]: y
acme
Successfully removed /var/home/stone/Clients/ACME
```

Running the command without `--rm-home` should ignore the home directory and only delete the container.
